src/events/dto/users-added.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { AddUsersToChannelDto } from 'src/channels/dto/add-users-to-channel.dto';
+import { BroadcastDto, EventDto } from './event.dto';
+import { Channel } from 'src/channels/domain/channel';
+import { IsNotEmpty, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+import { Workspace } from 'src/workspaces/domain/workspace';
+
+class BroadcastUsersAddedDto extends BroadcastDto {
+  @IsNotEmpty()
+  channel_id: Channel['id'];
+
+  @IsNotEmpty()
+  workspace_id: Workspace['id'];
+}
```

Comment:
If we need the workspace to perform adding users to channels, not in broadcasting, I think it must be in the `AddUsersToChannelDto`